django_switchuser/state.py

```diff
@@ -11,6 +11,8 @@ def __init__(self, request):
         self._reset()
 
     def _reset(self):
+        User = get_user_model()
```

Comment:
So we have a custom user model in Django and to maintain the functionality that we have with switchusers, we need to return the user model that we defined as opposed to the default `django.contrib.auth.User` model.